src/services/storage/drivers/inMemory/index.ts

```diff
@@ -1,8 +1,17 @@
 import cryptoRandomString from 'crypto-random-string'
 import { NotFoundError } from '../../../../errors/notFound.js'
+import { InvalidConfigError } from '../../../../errors/invalidConfig.js'
 import { InMemoryStorageConfig } from '../../types/config.js'
 import type { StorageDriver } from '../../types/index.js'
-import type { StoredUrl, UrlWithInformation, UrlRequestData, UrlInformation } from '../../types/url.js'
+import type {StoredUrl, UrlWithInformation, UrlRequestData, UrlInformation} from '../../types/url.js'
+import {logger} from "../../../logger/logger";
+import {getRawConfig} from "../../../../config/__test__/helpers";
```

Comment:
It's better to pass the expireFrom via the driver configurations instead of importing the config.

Also, you imported functions from the tests.